packages/core/src/service/services/collection.service.ts

```diff
@@ -431,7 +436,7 @@ export class CollectionService implements OnModuleInit {
     /**
      * Applies the CollectionFilters
      */
-    private async applyCollectionFiltersInternal(collection: Collection): Promise<ID[]> {
+    private async applyCollectionFiltersInternal(collection: Collection, applyToChangedVariantsOnly?: boolean): Promise<ID[]> {
```

Comment:
Can you add a short description of what "applyToChangedVariantsOnly" means and why we have it here? I will surely forget in the future and then need to dig through git history to get the context back!

Reply:
Added description